modules/nf-core/fastcov/meta.yml

```diff
@@ -0,0 +1,56 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: fastcov
+description: generate a coverage plot from one or more bam files
+keywords:
+  - coverage
+  - bam
+  - map
+tools:
+  - fastcov:
+      description: "generate a coverage plot from one or more bam files"
+      homepage: "https://github.com/RaverJay/fastcov/"
+      documentation: "https://github.com/RaverJay/fastcov/blob/main/README.md"
+      tool_dev_url: "https://github.com/RaverJay/fastcov/tree/main"
+      licence: ["MIT"]
+
+input:
```

Comment:
You need to update this meta.yml to the more recent structure that has been implemented since you created the PR, try running `nf-core modules lint fastcov --fix`.
Or look at other modules and update accordingly.

Reply:
Done :)